source/faq.txt

```diff
@@ -24,13 +24,25 @@ What Is the Difference Between "connectTimeoutMS", "socketTimeoutMS" and "maxTim
      - Description
    * - connectTimeoutMS
      - 30000
-     - ``connectTimeoutMS`` is a `connection-setting <https://mongodb.github.io/node-mongodb-native/3.5/reference/connecting/connection-settings/>`_ option that sets the time in milliseconds to establish a connection to the MongoDB server before timing out.
+     - ``connectTimeoutMS`` is a :doc:`connection option
+       </fundamentals/connection#connection-options>` that sets the time in
+       milliseconds for an individual connection from your connection pool to
+       establish a TCP connection to the MongoDB server before timing out.
+       Use the ``serverSelectionTimeoutMS`` option instead to fail when
```

Comment:
"This is not directly related to :node-api:``MongoClient.connect <MongoClient.html#.connect>`, if you wish to control the timeout for an initial connection to a cluster please see <link to `serverSelectionTimeoutMS` docs>"